swagger.yaml

```diff
@@ -417,6 +439,9 @@ definitions:
     properties:
       name:
         type: string
+      last_data_upload:
+        type: string
+        format: date-time
```

Comment:
What is the value of tracking when data sync occurred?

Reply:
So the admin/user would know whether data they have in the bucket are up to date. They may see old timestamps in the storage and files still can be up to date and the time in CR will confirm that.

Reply:
Once they have that knowledge what mechanisms do we plan for them to react if something is not right?

Reply:
If they have access to the device, they can check the device logs, see what's the problem is and fix it; most likely it would be network or remote storage problems.